dockerfiles/init/modules/compose/templates/docker-compose.yml.erb

```diff
@@ -24,6 +24,8 @@ che:
     - '/etc/passwd:/etc/passwd:ro'
 <% end -%>
   ports:
+    - '32001:32001'
```

Comment:
well I would say that it should be turned off by default (or an option to turn it off) as you may not require JMX at all